runtime/doc/options.txt

```diff
@@ -2803,7 +2803,9 @@ A jump table for the options with a short description can be found at |Q_op|.
 				:set guicursor=n:blinkon0
 <		{group-name}
 			a highlight group name, that sets the color and font
-			for the cursor
+			for the cursor. If an invalid group is set, the TUI won't change colors
+      contrary to vim. If the group has guibg=NONE then the cursor becomes
+      invisible.
```

Comment:
did you mean UI instead of TUI? (We can't control what GUIs do, but we can add to the guidelines at `:help dev-ui` which should be considered the "official UI behavior").

Can elide "contrary to vim".